packages/mason_cli/lib/src/commands/make.dart

```diff
@@ -85,6 +127,12 @@ class _MakeCommand extends MasonCommand {
     final target = DirectoryGeneratorTarget(Directory(outputDir));
     final disableHooks = results['no-hooks'] as bool;
     final quietMode = results['quiet'] as bool;
+
+    final watch = results['watch'] as bool;
+    if (watch && _brick.path == null) {
```

Comment:
@felangel when would a Brick not have a path to a BrickYaml? Hosted, Local or Global bricks all have a path?